iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/infospace/ImportInformationSpaceUtils.java

```diff
@@ -0,0 +1,30 @@
+package eu.dnetlib.iis.wf.importer.infospace;
+
+import org.apache.hadoop.mapreduce.JobContext;
+
+import eu.dnetlib.iis.common.WorkflowRuntimeParameters;
+
+/**
+ * Set of information space importer utilities.
+ * 
+ * @author mhorst
+ *
+ */
+public class ImportInformationSpaceUtils {
```

Comment:
I've moved it to `eu.dnetlib.iis.common.WorkflowRuntimeParameters` from which `UNDEFINED_NONEMPTY_VALUE` is references.